src/content/docs/apm/agents/php-agent/other-integrations/magento-integration.mdx

```diff
@@ -0,0 +1,254 @@
+        
+        To install the agent, follow the instructions in our [guided install](https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9&cards[0]=eyJuZXJkbGV0SWQiOiJzZXR1cC1uZXJkbGV0cy5zZXR1cC1waHAtaW50ZWdyYXRpb24iLCJhY2NvdW50SWQiOjI2NDA0MDl9&platform[accountId]=1).
+        
+        If you need to install the agent in a different way, see [PHP agent installation overview](/docs/apm/agents/php-agent/installation/php-agent-installation-overview/).
+
+        </Step>
+    
+    <Step>
+        ## Install our MySQL integration
+
+        Our MySQL integration monitors your databases and collects metrics such as database utilization and connections such as slowest queries and active connections by node.
+
+        To install the integration, follow the instructions in our guided install by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Add data > (search for MySQL) > (select an account)**.
+        
+        If you need to install the agent in a different way, see [MySQL monitoring integration](/install/mysql/).
+
+        </Step>
+    
+    <Step>
+        ## Install our Apache integration
+
+        Our Apache integrations collects network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker status.
+
+        To install the integration, Follow the instructions in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
+        
+        If you need to install the agent in a different way, see [Apache monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration).
+
+        </Step>
+    
+
+    <Step>
+        ## Install our Elasticsearch integration
+        
+        Our Elasticsearch integration collects metrics like cluster status and current shards. If you're using [Magento version 2.4 and higher](https://magentip.com/install-magento-2-4-x-on-ubuntu-with-elasticsearch/), you'll to install an ElasticSearch application and agent to monitor that application. 
+        
+        To install the integration, follow the instructions in our guided install by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Add data > (search for Elasticsearch) > (select an account)**.
+
+        If you need to install the agent in a different way, see [Apache monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-integration).
+
+        </Step>
+    
+
+    <Step>
+        ## Create synthetic monitors
```

Comment:
Is this optional or required?

Reply:
It's required in order to capture the server downtime.